node/libs/roles/src/proto/attester.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package zksync.roles.attester;
+
+import "zksync/std.proto";
```

Comment:
remove if your are not using anything from std.proto

Reply:
it is still here :)

Reply:
Sorry, fixed!